plugins/generator-1.21.1/neoforge-1.21.1/templates/plant/plant.java.ftl

```diff
@@ -411,7 +411,13 @@ public class ${name}Block extends ${getPlantClass(data.plantType)}Block
 <#macro canPlaceOnList blockList condition>
 	<#if (blockList?size > 1) && condition>(</#if>
 	<#list blockList as canBePlacedOn>
-	groundState.is(${mappedBlockToBlock(canBePlacedOn)})<#sep>||
+	<#if canBePlacedOn.getUnmappedValue()?starts_with("TAG:")>

```

Comment:
In other cases like this one, we use ".startsWith" instead of ?, so I suggest to also use it here for sake of easier searching through the codebase.

![image](https://github.com/user-attachments/assets/e4ce234b-75b3-48a8-abbb-fe2170b965b2)